activiti-cloud-query-service/activiti-cloud-starter-query/src/test/resources/application.properties

```diff
@@ -33,3 +33,5 @@ keycloak.security-constraints[1].authRoles[0]=ACTIVITI_ADMIN
 keycloak.security-constraints[1].securityCollections[0].patterns[0]=/admin/*
 
 spring.jpa.properties.hibernate.generate_statistics=true
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.show-sql=true
```

Comment:
Was this a temporary change? I'm afraid this will increase a lot the size of the logs.

Reply:
It does not increase logs as long as the logger level is OFF for tests

Reply:
Ok, so we need to set it to OFF for Hibernate in the github actions because it's currently enabled https://github.com/Activiti/activiti-cloud/runs/4672967808?check_suite_focus=true#step:7:10434.